internal/plan/plan.go

```diff
@@ -711,8 +855,9 @@ func (l *Layer) checkCycles() error {
 
 func ParseLayer(order int, label string, data []byte) (*Layer, error) {
 	layer := Layer{
-		Services: map[string]*Service{},
-		Checks:   map[string]*Check{},
+		Services:   map[string]*Service{},
```

Comment:
Should we also check for duplicates in Services.LogTargets after parsing it, and error if there are dupes in a single layer? (I think for merging layers it shouldn't be an error be de-duping, as you've done.)